src/map/utils/charutils.cpp

```diff
@@ -4754,7 +4754,7 @@ namespace charutils
                 "pos_y = %.3f,"
                 "pos_z = %.3f,"
                 "boundary = %u "
-                "WHERE charid = %u AND pos_zone = %u;";
```

Comment:
What happens to the corresponding currZone it was passing into that query on line 4768?